src/layout/section/seaSection/SeaSection.style.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components'
+
+import { color, font, responsiveBreakpoints, space } from '../../../_utils/branding'
+import { TextSubHeader } from '../../../typography/subHeader'
+
+const CONTENT_WIDTH = '375px' // iPhone6 width same as componentSizes.searchOverlayWidth
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-areas:
+    'media'
+    'body';
+  grid-template-rows: minmax(33vh, 33vh) auto;
+  background-color: ${color.white};
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    height: 100vh; //@todo: strech to parent
```

Comment:
This is a new file but it has a `@todo`, could you do it in this PR?

Reply:
Infortunally I can't because **height** in CSS depends on the parent. I can't define the height for `body` or `root` since its part of `Layout` 🤷‍♀️

Reply:
ok then 
- [ ] remove comment